app/components/participant_status_tag_component.rb

```diff
@@ -1,9 +1,10 @@
 # frozen_string_literal: true
 
 class ParticipantStatusTagComponent < BaseComponent
-  def initialize(profile:, induction_record: nil)
+  def initialize(profile:, induction_record: nil, has_mentees: false)
```

Comment:
How about use the ParticipantProfile::Mentor `has_mentees` relationship instead?
i.e. `profile.has_mentees.any?`

Reply:
makes sense